tests/test_posts.py

```diff
@@ -0,0 +1,73 @@
+import difflib
+from collections import defaultdict
+import glob
+
+from base import BaseTest
+
+import frontmatter
+
+
+class TestDefis(BaseTest):
```

Comment:
Class name changed in https://github.com/entrepreneur-interet-general/site-eig/commit/bf95a5dd7cebbd082b88472101320d21d071c6b3